src/main/java/com/bearAndPupperCo/sangenWrestlingApp/Services/WrestlerServiceImpl.java

```diff
@@ -1,106 +1,62 @@
 package com.bearAndPupperCo.sangenWrestlingApp.Services;
 
-import com.bearAndPupperCo.sangenWrestlingApp.DTO.WrestlerDTO;
+import com.bearAndPupperCo.sangenWrestlingApp.APIUtils.ValidationUtils;
+import com.bearAndPupperCo.sangenWrestlingApp.DTO.WrestlerMainTableDTO;
 import com.bearAndPupperCo.sangenWrestlingApp.Entities.Wrestler;
-import com.bearAndPupperCo.sangenWrestlingApp.Entities.WrestlingTitle;
-import com.bearAndPupperCo.sangenWrestlingApp.Exception.WrestlerAlreadyExistsException;
+import com.bearAndPupperCo.sangenWrestlingApp.Enum.WrestlerMainTableEnum;
+import com.bearAndPupperCo.sangenWrestlingApp.Exception.WrongParamException;
 import com.bearAndPupperCo.sangenWrestlingApp.Pagination.PaginatedResponse;
 import com.bearAndPupperCo.sangenWrestlingApp.Repository.WrestlerRepo;
-import com.bearAndPupperCo.sangenWrestlingApp.Repository.WrestlingTitleRepo;
 import org.modelmapper.ModelMapper;
-import org.modelmapper.TypeMap;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.domain.Page;
-import org.springframework.data.domain.PageRequest;
-import org.springframework.data.domain.Pageable;
-import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
-import java.util.stream.Collectors;
+
+import static com.bearAndPupperCo.sangenWrestlingApp.APIUtils.MessageConstants.WRONG_PARAM_MSG;
 
 @Service
 public class WrestlerServiceImpl implements WrestlerSrv{
 
     private final WrestlerRepo wrestlerRepo;
+    private final ValidationUtils validationUtils;
 
-    private final ModelMapper modelMapper;
-
-    public WrestlerServiceImpl(WrestlerRepo wrestlerRepo, ModelMapper modelMapper) {
+    public WrestlerServiceImpl(WrestlerRepo wrestlerRepo, ValidationUtils validationUtils) {
         this.wrestlerRepo = wrestlerRepo;
-        this.modelMapper = modelMapper;
-        initializeTypeMap();
-    }
-
-    private void initializeTypeMap() {
-        TypeMap<Wrestler, WrestlerDTO> propertyMapper = modelMapper.createTypeMap(Wrestler.class, WrestlerDTO.class);
-        modelMapper.getConfiguration().setSkipNullEnabled(true);
-        modelMapper.getConfiguration().setPreferNestedProperties(false);
-        defineWrestlerMapper(propertyMapper);
+        this.validationUtils = validationUtils;
     }
 
     @Override
     public Wrestler addNewWrestler(Wrestler wrestler) {
-        verifyIfWrestlerExists(wrestler);
+        //verifyIfWrestlerExists(wrestler);
```

Comment:
add TODO to be able to find - maybe even associate an issue/task number if already existing

Reply:
Done!